packages/components/src/local/organisms/setting-forces/settings-force-roles/index.tsx

```diff
@@ -98,6 +98,7 @@ export const RolesAccordion: FC<PropTypes> = ({ data, handleChangeForce }) => {
 
   const handleCreateRole = (): void => {
     const roles: Array<Role> = [...data.roles, {
+      roleId: 'r' + Math.random().toString(36).substring(4), // make id more uniq
```

Comment:
@lebaphi - elsewhere in the app we use the `uniqid` library: https://github.com/adamhalasz/uniqid/

The above would be replaced with `uniqid.time('r')`